doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -773,13 +811,14 @@ async def sync_payloads(self):
 
         await asyncio.sleep(120)
 
-        # Update the imagestreams being monitored by the release controller.
-        tasks = []
-        for arch, payload_entries in self.payload_entries_for_arch.items():
-            for private_mode in self.privacy_modes:
+        # Updating public and private image streams
+        for private_mode, payload_entries_for_each_arch_iter in [(False, self.payload_entries_for_arch), (True, self.private_payload_entries_for_arch)]:
```

Comment:
let's rename the attr self.payload_entries_for_arch -> self.public_payload_entries_for_each
so it's crystal clear that these are supposed to be only public entries